airflow/dags/sandbox/load_airtable_child.yml

```diff
@@ -1,11 +1,13 @@
 operator: operators.AirtableToWarehouseOperator
 
 air_base_id: apphcwzP8cR82NeJo
+air_base_name: sandbox
 air_table_name: child
-table_name: sandbox.airtable_child
+bucket: "gs://{{ prefix_bucket('airtable') }}"
```

Comment:
do both test-airtable and airtable buckets exist yet?

Reply:
Good call, `airtable` was taken (I forgot bucket names have to be *globally* unique 🥴). Switched both to `calitp-airtable` which I did create (test & prod).